src/i18n/translations/en/translation.json

```diff
@@ -47,10 +47,10 @@
     "popularDocs": {
       "title": "Popular docs",
       "t1": {
-        "title": "NRQL syntax, clauses, and functions"
+        "title": "Monitor all your apps and services"
       },
       "t2": {
-        "title": "Forward logs using infrastructure agent"
+        "title": "Automatically identify and remediate vulnerabilities with Vulnerability Management"
```

Comment:
This is actually a bit wordy now that I think about it. If you can come up with something punchier that might be nice but we can go ahead and get htis out and update later I think